glean-core/ios/GleanTests/Scheduler/MetricsPingSchedulerTests.swift

```diff
@@ -164,7 +168,11 @@ class MetricsPingSchedulerTests: XCTestCase {
         )
         let expectedValue = "must-exist-in-the-first-ping"
 
+        expectation = setUpDummyStubAndExpectation(testCase: self, tag: "MetricsPingSchedulerTests")
         Glean.shared.resetGlean(clearStores: false)
+        waitForExpectations(timeout: 5.0) { error in
```

Comment:
Which ping do we expect here? As per the code above we should already delay the initial metrics ping, so what is sent?

Reply:
The baseline ping with reason=active.